ema_workbench/analysis/prim.py

```diff
@@ -978,11 +978,11 @@ def __init__(
             pass
         x = x.reset_index(drop=True)
 
-        x_float = x.select_dtypes([np.float32, np.float64, float])
+        x_float = x.select_dtypes([float, np.float64])
```

Comment:
Please look carefully if here and on line 985 the intendent behaviour preserved.